src/modules/GameConstants.ts

```diff
@@ -1200,3 +1202,50 @@ export enum DayOfWeek {
     'Friday',
     'Saturday',
 }
+
+// Roamer groups
+export enum RoamerGroupsKanto {
+    Kanto = 0,
+    Sevii123,
+}
+
+export enum RoamerGroupsJohto {
+    All = 0,
+}
+
+export enum RoamerGroupsHoenn {
+    All = 0,
+}
+
+export enum RoamerGroupsSinnoh {
+    All = 0,
+}
+
+export enum RoamerGroupsUnova {
+    All = 0,
+}
+
+export enum RoamerGroupsKalos {
+    All = 0,
+}
+
+export enum RoamerGroupsAlola {
+    All = 0,
+}
+
+export enum RoamerGroupsGalar {
+    Galar = 0,
+    IsleofArmor,
+    CrownTundra,
+}
+
+export const RoamerGroups = [
+    RoamerGroupsKanto,
+    RoamerGroupsJohto,
+    RoamerGroupsHoenn,
+    RoamerGroupsSinnoh,
+    RoamerGroupsUnova,
+    RoamerGroupsKalos,
+    RoamerGroupsAlola,
+    RoamerGroupsGalar,
+];
```

Comment:
Not sure how I feel about the naming of this..
Why not just have these as subregion, as it's essentially the same thing, but makes sense when it's used in other contexts too.

Reply:
Will also mean we don't need to change multiple places whenever a new subregion is added.
As these values are currently mostly duplicated here `src/modules/subRegion/SubRegions.ts`.

Reply:
Galar is split into north and south in the galar PR. The first Galar roamer group will cover both north and south subregion.
And Alola has 3 subregions, but only one roamer group.
I am not sure how i should use subregions here?

Reply:
The gameconstant enums are now the subregions, and roamergroups are now defined in the roamerlist